framework/source/class/qx/ui/table/Table.js

```diff
@@ -1378,8 +1378,8 @@ qx.Class.define("qx.ui.table.Table",
      */
     _onKeyPress : function(evt)
     {
- -     qx.log.Logger.deprecatedMethodWarning(arguments.callee,"The method '_onKeyPress()' is deprecated. Please use '_onKeyDown()' instead.");
- -     qx.log.Logger.deprecateMethodOverriding(this, qx.ui.table.Table, "_onKeyPress", "The method '_onKeyPress()' is deprecated. Please use '_onKeyDown()' instead.");
+       qx.log.Logger.deprecatedMethodWarning(qx.ui.table.Table._onKeyPress, "The method '_onKeyPress()' is deprecated. Please use '_onKeyDown()' instead.");
```

Comment:
That call does not work as intended!
As `_onKeyPress` isn't a static function/method of `qx.ui.table.Table`.
So either use `qx.ui.table.Table.prototype._onKeyPress` or the much simpler `this._onKeyPress` 😄 

I think a simple `this._onKeyPress` would be the correct choice here:
`qx.log.Logger.deprecatedMethodWarning(this._onKeyPress, "The method '_onKeyPress()' is deprecated. Please use '_onKeyDown()' instead.");`

Reply:
could you create an PR? Thanks!

Reply:
Done: #9559